examples/rollup/requirements.txt

```diff
@@ -1,12 +1,12 @@
 certifi==2022.12.7
```

Comment:
Is this related?

Reply:
These version bumps are required for the rollup example to compile